### PR TITLE
feat: buttons added

### PR DESCRIPTION
**Pull request by sakshamAK**

### CODE REVIEW

**Comment by poornima-alugubelly** [comment]
Looks good Saksham!

**Review by kshitijthatte — APPROVED** [review]
Looks good, try adding styles for the hover state.

**Comment by SJTGSHIVAM** [comment]
Looks good to me.

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        display: none;
+    }
+
+    .links li {
+        padding: 0;
+    }
+    .nav-title .material-icons{
+        display: none;
+    }
+    .links li a {
```
Comment (SJTGSHIVAM) [inline components/button/button.css:117]: links seems to be a very generic name you can improve on variable naming in the future but for now this looks good

**Comment by sakshamAK** [comment]
> Looks good, try adding styles for the hover state.

Yup sure, thanks!

File: components/button/button.css (hunk 1 above)
Comment (sakshamAK) [reply components/button/button.css:117]: sure, thanks!

**Pull request merged**